net/picotls/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+PortGroup           openssl 1.0
+
+# O_CLOEXEC
+legacysupport.newest_darwin_requires_legacy 10
+
+github.setup        h2o picotls df13092f6c5c2f08c5227e096a1590603229dfc8
+version             2023.12.04
+revision            0
+categories          net security
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         &{name} is a TLS 1.3 (RFC 8446) protocol stack written in C
+long_description    {*}${description}. &{name} is designed to be fast, tiny and low-latency, \
+                    with the primary user being the H2O HTTP/2 server for serving \
+                    HTTP/1, HTTP/2 and HTTP/3 over QUIC.
+
+fetch.type          git
+
+if {${os.platform} eq "darwin" && ${os.major} < 14} {
+    # Lion+ (with Xcode 4.1+) have git; earlier need to bring their own.
+    # However Lion through Mavericks fail with SSL errors.
+    depends_build-append \
+                    port:git
+    git.cmd         ${prefix}/bin/git
+}
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
+}
```

Comment:
The only dependency which is uses is outdated https://github.com/h2o/picotest; maybe include it into distfiles and avoid this?

Reply:
Fair enough. I will try that.